tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Content/ContentWithMultiURLPicker.spec.ts

```diff
@@ -0,0 +1,265 @@
+});
+
+test('can publish content with the document link', async ({umbracoApi, umbracoUi}) => {
+  // Arrange
+  const dataTypeData = await umbracoApi.dataType.getByName(dataTypeName);
+  await umbracoApi.documentType.createDocumentTypeWithPropertyEditor(documentTypeName, dataTypeName, dataTypeData.id);
+  // Create a document to link
+  const dtForLinkedDocumentName = 'TestDocumentType';
+  const dtForLinkedDocumentId = await umbracoApi.documentType.createDefaultDocumentTypeWithAllowAsRoot(dtForLinkedDocumentName);
+  const linkedDocumentName = 'ContentToPick';
+  const linkedDocumentId = await umbracoApi.document.createDefaultDocument(linkedDocumentName, dtForLinkedDocumentId);
+  await umbracoUi.content.goToSection(ConstantHelper.sections.content);
+
+  // Act
+  await umbracoUi.content.clickActionsMenuAtRoot();
+  await umbracoUi.content.clickCreateButton();
+  await umbracoUi.content.chooseDocumentType(documentTypeName);
+  await umbracoUi.content.enterContentName(contentName);
+  await umbracoUi.content.clickAddMultiURLPickerButton();
+  await umbracoUi.content.selectLinkByName(linkedDocumentName);
+  await umbracoUi.content.clickSubmitButton();
+  await umbracoUi.content.clickSaveAndPublishButton();
+
+  // Assert
+  await umbracoUi.content.doesSuccessNotificationsHaveCount(2);
+  expect(await umbracoApi.document.doesNameExist(contentName)).toBeTruthy();
+  const contentData = await umbracoApi.document.getByName(contentName);
+  expect(contentData.variants[0].state).toBe('Published');
+  expect(contentData.values[0].alias).toEqual(AliasHelper.toAlias(dataTypeName));
+  expect(contentData.values[0].value.length).toBe(1);
+  expect(contentData.values[0].value[0].type).toEqual('document');
+  expect(contentData.values[0].value[0].icon).toEqual('icon-document');
+  expect(contentData.values[0].value[0].target).toBeNull();
+  expect(contentData.values[0].value[0].unique).toEqual(linkedDocumentId);
+  // Uncomment this when the front-end is ready. Currently the link title is not auto filled after choosing document to link
+  //expect(contentData.values[0].value[0].name).toEqual(linkedDocumentId);
+
+  // Clean
+  await umbracoApi.documentType.ensureNameNotExists(dtForLinkedDocumentName);
+  await umbracoApi.document.ensureNameNotExists(linkedDocumentName);
+});
+
+test('can create content with the external link', async ({umbracoApi, umbracoUi}) => {
+  // Arrange
+  const link = 'https://docs.umbraco.com';
+  const linkTitle = 'Umbraco Documentation';
+  const dataTypeData = await umbracoApi.dataType.getByName(dataTypeName);
+  await umbracoApi.documentType.createDocumentTypeWithPropertyEditor(documentTypeName, dataTypeName, dataTypeData.id);
+  await umbracoUi.content.goToSection(ConstantHelper.sections.content);
+
+  // Act
+  await umbracoUi.content.clickActionsMenuAtRoot();
+  await umbracoUi.content.clickCreateButton();
+  await umbracoUi.content.chooseDocumentType(documentTypeName);
+  await umbracoUi.content.enterContentName(contentName);
```

Comment:
Instead of doing this in each test, we can create content with the multiUrlPicker in the API and the go to it